models/volunteer.ts

```diff
@@ -0,0 +1,15 @@
+import type { provinces, locales } from "@prisma/client";
+
+export type CreateVolunteerInput = {
```

Comment:
same point as above - can we use this to return from the GET endpoint?

Reply:
see discussion https://github.com/uwblueprint/social-diversity-for-children/pull/18#discussion_r659917321